docs/docs/docs/01-core/data-transfer/01-engine.md

```diff
@@ -197,9 +197,27 @@ The following events are available:
 
 ### Diagnostics events
 
-`engine.diagnostics.onDiagnostic(formatDiagnostic('export'));`
+The engine includes a diagnostics reporter which can be used to listen for diagnostics information (debug messages, errors, etc).
```

Comment:
Maybe we could show example on how to listen on specific events (onError, onWarning, onInfo)